modules/module1/slides/module1_03.qmd

```diff
@@ -0,0 +1,146 @@
+---
+format: revealjs
+title: Types of machine learning
+title-slide-attributes:
+  data-notes: |
+    This course is about supervised machine learning. In fact, all the examples we saw in the last section are examples of supervised machine learning, but what other types of machine learning are there?
+
+---
+
+```{python}
+#  | echo: false
+%run src/utils.py
+```
+
+## Typical learning problems:  
+
+- **Supervised learning (this course)** (<a href="https://support.google.com/a/answer/2368132?hl=en" target="_blank">Gmail spam filtering</a>)
+    - Training a model from input data and its corresponding labels to predict new examples.     
+- Unsupervised learning (<a href="https://news.google.com/" target="_blank">Google News</a>)
+    - Training a model to find patterns in a dataset, typically an unlabeled dataset.
+- Reinforcement learning (<a href="https://deepmind.com/research/case-studies/alphago-the-story-so-far" target="_blank">AlphaGo</a>)
+    - A family of algorithms for finding suitable actions to take in a given situation in order to maximize a reward. 
+- Recommendation systems (<a href="https://www.cs.umd.edu/~samir/498/Amazon-Recommendations.pdf" target="_blank">Amazon item recommendation system</a>)
+    - Predict the "rating" or "preference" a user would give to an item.    
+
+:::{.notes}
+
+Some typical types of learning problem include:<br><br>
```

Comment:
```suggestion
Some typical types of learning problem include:<br><br>

```

These notes are not rendering as bullet point, I wonder if it is because there is a missing newline here. Otherwise try to write it out as an HTML list.